src/goto-symex/goto_state.cpp

```diff
@@ -10,6 +10,7 @@ Author: Romain Brenguier, romain.brenguier@diffblue.com
 #include "goto_symex_is_constant.h"
 #include "goto_symex_state.h"
 
+#include <util/arith_tools.h>
```

Comment:
Is this used? (After a quick scan I couldn't see an obvious use)

Reply:
You're right, I think I must have added it for `to_integer` in an earlier version, but I'm not using that any more. I've removed it.